android/modules/locale/src/java/ti/modules/titanium/locale/LocaleModule.java

```diff
@@ -21,8 +23,16 @@
 import org.appcelerator.titanium.util.TiPlatformHelper;
 import org.appcelerator.titanium.util.TiRHelper;
 
+import android.os.Build;
 import android.telephony.PhoneNumberUtils;
 
+import androidx.appcompat.app.AppCompatDelegate;
+import androidx.appcompat.app.AppCompatDelegate;
+import androidx.appcompat.app.AppCompatDelegate;
+import androidx.appcompat.app.AppCompatDelegate;
+import androidx.appcompat.app.AppCompatDelegate;
```

Comment:
```suggestion
import androidx.appcompat.app.AppCompatDelegate;
```

strange, it added it multiple times. I'll fix that